scripts/ci/build_emscripten.sh

```diff
@@ -40,13 +40,14 @@ else
 	BUILD_DIR="$1"
 fi
 
-apt-get update
-apt-get install lz4 --no-install-recommends
-
 WORKSPACE=/root/project
 
 cd $WORKSPACE
 
+# NOTE: workaround to git unsafe repository error by adding /root/project as a safe.directory
+# See https://github.blog/2022-04-12-git-security-vulnerability-announced/
```

Comment:
Would be better if the comment explained why we need to do this:
```suggestion
# NOTE: Without `safe.directory` git would assume it's not safe to operate on /root/project since it's owned by a different user.
# See https://github.blog/2022-04-12-git-security-vulnerability-announced/
```